test-harness/src/test/java/io/jenkins/plugins/casc/YamlMaxAliasesCollection.java

```diff
@@ -27,14 +26,14 @@
     @Test
     public void testAMaxOfOneEnv() {
         env.set(ConfigurationContext.CASC_YAML_MAX_ALIASES_ENV, "1");
-        assertThrows(YAMLException.class, () ->
+        assertThrows(ConfiguratorException.class, () ->
```

Comment:
Worth checking the message here?

Reply:
`assertThrows` does not allow for asserting the exception message.

Reply:
It returns the exception and then you can assert on it

Reply:
Oh it returns the exception so sure 😅

Reply:
The stack trace somewhat hides it but it is still visible.
```
io.jenkins.plugins.casc.ConfiguratorException: Failed to read YamlSource: file:/C:/git/code/configuration.as.code.plugin/test-harness/target/test-classes/io/jenkins/plugins/casc/maxAliasesLimit.yml
	at io.jenkins.plugins.casc.yaml.YamlUtils.merge(YamlUtils.java:48)
	at io.jenkins.plugins.casc.yaml.YamlUtils.loadFrom(YamlUtils.java:127)
	at io.jenkins.plugins.casc.ConfigurationAsCode.configureWith(ConfigurationAsCode.java:618)
	at io.jenkins.plugins.casc.ConfigurationAsCode.configure(ConfigurationAsCode.java:587)
	at io.jenkins.plugins.casc.ConfigurationAsCode.configure(ConfigurationAsCode.java:576)
	at io.jenkins.plugins.casc.YamlMaxAliasesCollection.lambda$testAMaxOfOneProp$1(YamlMaxAliasesCollection.java:42)
	at org.junit.Assert.assertThrows(Assert.java:1001)
	at org.junit.Assert.assertThrows(Assert.java:981)
	at io.jenkins.plugins.casc.YamlMaxAliasesCollection.testAMaxOfOneProp(YamlMaxAliasesCollection.java:40)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:498)
	at org.junit.runners.model.FrameworkMethod$1.runReflectiveCall(FrameworkMethod.java:59)
	at org.junit.internal.runners.model.ReflectiveCallable.run(ReflectiveCallable.java:12)
	at org.junit.runners.model.FrameworkMethod.invokeExplosively(FrameworkMethod.java:56)
	at org.junit.internal.runners.statements.InvokeMethod.evaluate(InvokeMethod.java:17)
	at org.jvnet.hudson.test.JenkinsRule$1.evaluate(JenkinsRule.java:597)
	at org.junit.internal.runners.statements.FailOnTimeout$CallableStatement.call(FailOnTimeout.java:288)
	at org.junit.internal.runners.statements.FailOnTimeout$CallableStatement.call(FailOnTimeout.java:282)
	at java.util.concurrent.FutureTask.run(FutureTask.java:266)
	at java.lang.Thread.run(Thread.java:748)
Caused by: io.jenkins.plugins.casc.ConfiguratorException: Number of aliases for non-scalar nodes exceeds the specified max=1
You can increase the maximum by setting an environment variable or property
  ENV: CASC_YAML_MAX_ALIASES="100"
  PROPERTY: -Dcasc.yaml.max.aliases="100"
	at io.jenkins.plugins.casc.yaml.YamlUtils.read(YamlUtils.java:66)
	at io.jenkins.plugins.casc.yaml.YamlUtils.merge(YamlUtils.java:38)
	... 21 more

```